units/swap-create@.service

```diff
@@ -2,13 +2,10 @@
 
 [Unit]
 Description=Create swap on /dev/%i
-Wants=systemd-modules-load.service
-After=systemd-modules-load.service
```

Comment:
Dropping `ExecStartPre=-modprobe` is OK. But please keep this part. We generally still prefer to load modules via `systemd-modules-load.service` then ourselves.

Reply:
https://github.com/systemd/zram-generator/issues/26: `modules-load.d` cannot be used, so ordering oneself after it seems foolish; marking resolved.